sys-kernel/bliss-initramfs/metadata.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+    <maintainer type="person">
+        <email>jon@xyinn.org</email>
```

Comment:
just add your previously used @gentoo.org addrr

Reply:
fixing, I didn't add fearedbliss@g.o originally since my email is not active yet ;D.